aten/src/ATen/native/quantized/cpu/fbgemm_utils.cpp

```diff
@@ -197,3 +205,65 @@ Tensor ConvertToChannelsLast3dTensor(const Tensor& src) {
 } // namespace at
 
 #endif // USE_FBGEMM
+
+
+torch::jit::class_<LinearPackedParamsBase> register_linear_params() {
+  using SerializationType = std::tuple<at::Tensor, c10::optional<at::Tensor>>;
+  static auto register_linear_params =
+      torch::jit::class_<LinearPackedParamsBase>("quantized", "LinearPackedParamsBase")
```

Comment:
This can be put into the `TORCH_LIBRARY` block now. Just say:

```
m.class_<LinearPackedParamsBase>("LinearPackedParamsBase")
  ...
```

LMK if dependency ordering means you can't conveniently put this in the other C++ file.